### PR TITLE
Add Example for sending information to DB via UDP

### PR DESCRIPTION
**Pull request by shantanoo-desai**
Due to lack of documentation for UDP, this example
 provides basic usage of sending information points
 via UDP. The code structure followed is similar, if
 not same as other examples in the `examples` directory.

 Locally tested with:
 __InfluxDB v1.6.1__ on Windows10 and __influxdb-python v5.0.0__
 and should not break anything

### Things To do

- [x] Update Documentation

This PR addresses #646 #647 

Signed-off-by: Shantanoo <shantanoo.desai@gmail.com>

### CODE REVIEW

**Comment by shantanoo-desai** [comment]
@aviau  Any suggestions on how to overcome the failed tests?

**Comment by shantanoo-desai** [comment]
@aviau  Any suggestions on how to overcome the failed tests?

**Comment by shantanoo-desai** [comment]
@aviau any suggestions on how to pass the CI checks?

**Comment by shantanoo-desai** [comment]
Should I change the `.travis.yml` file or will this be done by someone else?

**Comment by aviau** [comment]
Yes, someone should do it.

That would be in a separate pull request.

**Comment by nevmerzhitsky** [comment]
Please finish the task

**Comment by shantanoo-desai** [comment]
@nevmerzhitsky I had some problems with `tox` on my WSL and will complete it tomorrow

**Comment by shantanoo-desai** [comment]
@nevmerzhitsky could you provide me a tutorial or some help as to how do I update the documentation?

**Comment by shantanoo-desai** [comment]
@nevmerzhitsky @aviau @xginn8  Travis tests have passed on the documentations. I however want to address that during the `tox -e docs` I came across three warnings, where the `docstring` one core file (`client.py`) has some linting errors. 
And also a Warning `html_static_path` entry error which stems from `docs/sources/conf.py`.

I have already resolved the Issue and wanted to know if this should be on a different PR or should I push the code here?

File: examples/tutorial_udp.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""Example for sending batch information to InfluxDB via UDP."""
+
+"""
+INFO: In order to use UDP, one should enable the UDP service from the
+`influxdb.conf` under section
+    [[udp]]
+        enabled = true
+        bind-address = ":8089" # port number for sending data via UDP
```
Comment (xginn8) [inline examples/tutorial_udp.py]: I think this value should be consistent with the port used in the default example code.

Comment (xginn8) [inline examples/tutorial_udp.py]: I think this value should be consistent with the port used in the default example code.

Comment (shantanoo-desai) [reply examples/tutorial_udp.py]: According to [Docs](https://docs.influxdata.com/influxdb/v1.7/supported_protocols/udp/) the default UDP Port is `8089` @xginn8

Comment (shantanoo-desai) [reply examples/tutorial_udp.py]: The UDP port by default configuration is 8089. Port 8086 is required to connect to the DB.

**Comment by shantanoo-desai** [comment]
@xginn8 The changes you requested were already addressed in the PR. The ports for UDP is `8089` which is by default and also according to the Docs. Everything else in the code is in accordance with the other examples.

**Comment by xginn8** [comment]
> @xginn8 The changes you requested were already addressed in the PR. The ports for UDP is `8089` which is by default and also according to the Docs. Everything else in the code is in accordance with the other examples.

@shantanoo-desai I think it makes more sense to pass the UDP port as an arg, rather than the TCP port used to connect to InfluxDB. You can omit the hostname and TCP port altogether if you want to make it more clear.

Also, please push any other fixes unrelated to this example in a separate PR.

**Comment by shantanoo-desai** [comment]
@xginn8 As mentioned, I have improved the `tutorial_udp.py`  file. All tests have passed. I mentioned the changes where I pass `udp_port` information as Argument for the example.

**Comment by xginn8** [comment]
thanks for contributing @shantanoo-desai!!

**Pull request merged**